src/main/routes/logout.ts

```diff
@@ -0,0 +1,30 @@
+import * as express from 'express';
+
+import { Paths } from 'main/app/paths';
+import { JwtExtractor } from 'idam/jwtExtractor';
+import { User } from 'idam/user';
+import { IdamClient } from 'idam/idamClient';
+import { Logger } from '@hmcts/nodejs-logging';
+
+import config from 'config';
```

Comment:
Minor but what do you think about keeping node modules above modules we author?

Reply:
I changed the order for now. Personally i only care about grouping on import specific, import * or import {} as i have done and leave the rest for intellij auto import.